addons/mail/static/src/components/messaging_menu/messaging_menu.js

```diff
@@ -167,6 +166,14 @@ class MessagingMenu extends Component {
     _onClickToggler(ev) {
         // avoid following dummy href
         ev.preventDefault();
+        if (!this.env.messaging) {
```

Comment:
unresolved comment

Reply:
will do

Reply:
Good catch on this, yes it's technically possible to have a crash in such case.

Fixed this issue by turning auto-loading of inbox main cache messages into a computed field (so that it can trigger loading exactly when inbox exists and all other conditions match).

To make test: have messaging created but session is not bound... that super specific with current implementation... the fact that it waits for session bound **after** creation of messaging is kinda weird... At least, doesn't look too complicated to simulate this case.

Reply:
moved `await session.is_bound` before messaging creation, so that there's no such 3rd problematic state